python/setup.py

```diff
@@ -94,9 +94,10 @@ def find_pybind_headers_dir():
 if len([fn for fn in os.listdir(NGRAPH_CPP_LIBRARY_DIR) if re.search("ngraphd", fn)]):
     NGRAPH_CPP_LIBRARY_NAME = "ngraphd"
 
-ONNX_IMPORTER_CPP_LIBRARY_NAME = "onnx_importer"
-if len([fn for fn in os.listdir(NGRAPH_CPP_LIBRARY_DIR) if re.search("onnx_importerd", fn)]):
-    ONNX_IMPORTER_CPP_LIBRARY_NAME = "onnx_importerd"
+if NGRAPH_ONNX_IMPORT_ENABLE in ["TRUE", "ON", True]:
```

Comment:
why give 3 ways to do this?

Reply:
This is my personal github account - was logged in for class assignment!

Reply:
Copied from line 239. Will check if we still need a 3-way check. Seems like the code is from the days when bool type was not normalized.